packages/stylelint-config/__tests__/wp-custom-property/style.css

```diff
@@ -0,0 +1,3 @@
+:root {
```

Comment:
@Antonio-Laguna @fabiankaegy added a basic test, could you add all of the variations of the rule we shall support?

it would be great to in the future do more unit testing of our config.

Reply:
Done